README.md

```diff
@@ -25,9 +25,13 @@ are gladly accepted!
 Setup
 =====
 
-Make sure you have jekyll installed (`gem install jekyll`), and run:
+Install the required dependencies using [Bundler](https://github.com/bundler/bundler)
```

Comment:
As I understand, using bundler is mostly for heroku here. For human
contributors, wouldn't it be simpler to keep using RubyGems only? Or
maybe have two sections (keep setup instructions with RubyGems, and
add those for Bundler).

Reply:
No, people should be using Bundler so that everyone working on the guides is using the same version of all the dependencies.

I realize that i would need to update `.travis.yml` and a few other bits of documentation as well, which i will add in another PR once this is merged.

Reply:
OK, thanks for your explanation.